src/main/java/com/bunq/sdk/model/generated/PaymentBatch.java

```diff
@@ -106,14 +106,19 @@
     return list(apiContext, userId, monetaryAccountId, new HashMap<>());
   }
 
+  public static BunqResponse<List<PaymentBatch>> list(ApiContext apiContext, Integer userId,
+      Integer monetaryAccountId, Map<String, String> params) {
+    return list(apiContext, userId, monetaryAccountId, params, new HashMap<>());
+  }
+
   /**
    * Return all the payment batches for a monetary account.
```

Comment:
Why sudanly a space between monetary and account ?

Reply:
generated

Reply:
suddenly* :D